rclcpp/test/rclcpp/test_create_timer.cpp

```diff
@@ -18,10 +18,10 @@
 #include <chrono>
 #include <memory>
 
+#include "node_interfaces/node_wrapper.hpp"
```

Comment:
nit: why this change is needed?

Reply:
It puts the includes in alphabetical order, which admittedly is unrelated to this PR, but the whole effort falls under "Increasing test coverage of rclcpp". Once this test is added, there then create_timer.hpp is fully covered.